faq.adoc

```diff
@@ -0,0 +1,76 @@
+<a href="url"><img src="https://d15n4q3o4x3svq.cloudfront.net/assets/tutorials/curl/api-a397cc184c5622fb5130af1b7baf149d.png" height="150" width="150" ></a>
+# Frequently Asked Questions
+This document contains answers to the most asked questions, if you have questions that are not answered
+in this document please refer to using our api-team email address: api@izettle.com .
+If you want to apply to receive access to our APIs, please fill in the form [here](https://docs.google.com/forms/d/e/1FAIpQLScdU9DDPfd680vhyZYAZ6DMIydQ1ikAOwP4abF2NWra5A-j8w/viewform)
+and we will come back to you with your credentials.
+
+---
+
+### General questions
+##### Question:
+> Does it cost anything to use your open APIs and is there some restriction on the API?
+
+##### Answer:
+
+> No, its completely free to use our API and the only limitation is that some resources and
+services might have rate-limiting enabled to prevent overload.
+
+___
+##### Question:
+> Do you provide any test environment accounts?
+
+##### Answer:
+> Unfortunately we do not at this time, there are however test accounts in production that you can use.
+This is something we are working on, the test accounts can be found [Here](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc).
+
+___
+##### Question:
+> Why do i receive `HTTP 100 Continue`  responses sometimes?
+
+##### Answer:
+
+> This is something our framework sends when it has received the request headers and
+indicates that the client should proceed to send the request body. What you can do is investigate
+ the reason why your code does this, or you could traverse the headers until
+you find a header with the intended response code, examples [Here](https://stackoverflow.com/questions/14526627/double-http-status-header-on-http-post-to-jersey)
+and [Here](https://stackoverflow.com/questions/2964687/how-to-handle-100-continue-http-message).
+
+___
+### [OAuth](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc)
+##### Question:
+> Once I have a Authorization token, how do I use it to authenticate myself?
```

Comment:
"Authorization token" should rather be named "Access token" to adhere to the OAuth specification.